docs/luigi-container.md

```diff
@@ -76,12 +76,12 @@ cd luigi/container
 ```bash
 npm install
 npm run build
-npm start
+npm run start-examples
 ```
 
 The app should be available at `http://localhost:8080` in your browser. 
 
-3. Check `./test-app/index.html` to see how Luigi Container is used.
+3. Check the examples folders at`./examples` to see how Luigi Container is used.
```

Comment:
./examples -> Make it a link to the examples folder, as in the previous comments above. 

The link for this should most likely be 
https://github.com/SAP/luigi/tree/main/container/examples 

Even though this link is broken for now, it should work after you merge your PR to main branch. Just put the /folder-name/file-name as needed and it should be okay.